### PR TITLE
chore(region-info): Add missing information for all supported regions

### PR DESCRIPTION
**Pull request by SankyRed**
Checks off some items from https://github.com/aws/aws-cdk/issues/26936

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts [hunk 1]
```diff
@@ -504,7 +505,7 @@ export const FIREHOSE_CIDR_BLOCKS: { [region: string]: string } = {
   'ap-northeast-2': '13.209.1.64',
   'ap-northeast-3': '13.208.177.192',
   'ap-south-1': '13.232.67.32',
-  'ap-south-2': '18.60.192.128',
+  'ap-south-2': '18.60.192.192',
```
Comment (rix0rrr) [inline packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts:508]: Is this on purpose?

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts [hunk 2]
```diff
@@ -521,7 +522,7 @@ export const FIREHOSE_CIDR_BLOCKS: { [region: string]: string } = {
   'eu-west-2': '18.130.1.96',
   'eu-west-3': '35.180.1.96',
   'il-central-1': '51.16.102.64',
-  'me-central-1': '3.28.159.32',
+  'me-central-1': '3.28.159.64',
```
Comment (rix0rrr) [inline packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts:525]: Is this on purpose?

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts (hunk 1 above)
Comment (SankyRed) [reply packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts]: Nice catch, when looked up the doc provided it does have Kinesis Data Firehose using one CIDR block for each available Region in different scenarios. Fixed them with the required values. Thank you!

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts [hunk 3]
```diff
@@ -4,25 +4,26 @@ export const AWS_CDK_METADATA = new Set([
...
   'eu-north-1',
   'eu-south-1',
-  // 'eu-south-2',
+  'eu-south-2',
   'eu-west-1',
   'eu-west-2',
   'eu-west-3',
   'il-central-1',
-  // 'me-central-1',
+  'me-central-1',
```
Comment (mrgrain) [inline packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts:26]: Revert, we have not deployed the metadata resource in this region

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts [hunk 4]
```diff
@@ -4,25 +4,26 @@ export const AWS_CDK_METADATA = new Set([
...
   'ca-central-1',
   'cn-north-1',
   'cn-northwest-1',
   'eu-central-1',
-  // 'eu-central-2',
+  'eu-central-2',
   'eu-north-1',
   'eu-south-1',
-  // 'eu-south-2',
+  'eu-south-2',
```
Comment (mrgrain) [inline packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts:21]: Revert, we have not deployed the metadata resource in this region

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts [hunk 5]
```diff
@@ -4,25 +4,26 @@ export const AWS_CDK_METADATA = new Set([
...
   'ap-southeast-1',
   'ap-southeast-2',
+  'ap-southeast-3',
   'ap-southeast-4',
   'ca-central-1',
   'cn-north-1',
   'cn-northwest-1',
   'eu-central-1',
-  // 'eu-central-2',
+  'eu-central-2',
```
Comment (mrgrain) [inline packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts:18]: Revert, we have not deployed the metadata resource in this region

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts [hunk 6]
```diff
@@ -4,25 +4,26 @@ export const AWS_CDK_METADATA = new Set([
...
   'ap-northeast-1',
   'ap-northeast-2',
-  // 'ap-northeast-3',
+  'ap-northeast-3',
   'ap-south-1',
-  // 'ap-south-2',
+  'ap-south-2',
   'ap-southeast-1',
   'ap-southeast-2',
+  'ap-southeast-3',
```
Comment (mrgrain) [inline packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts:12]: Comment out, we have not deployed the metadata resource in this region

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts [hunk 7]
```diff
@@ -4,25 +4,26 @@ export const AWS_CDK_METADATA = new Set([
   'ap-east-1',
   'ap-northeast-1',
   'ap-northeast-2',
-  // 'ap-northeast-3',
+  'ap-northeast-3',
   'ap-south-1',
-  // 'ap-south-2',
+  'ap-south-2',
```
Comment (mrgrain) [inline packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts:9]: Revert, we have not deployed the metadata resource in this region

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts [hunk 8]
```diff
@@ -4,25 +4,26 @@ export const AWS_CDK_METADATA = new Set([
   'ap-east-1',
   'ap-northeast-1',
   'ap-northeast-2',
-  // 'ap-northeast-3',
+  'ap-northeast-3',
```
Comment (mrgrain) [inline packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts:7]: Revert, we have not deployed the metadata resource in this region

**Review by mrgrain — CHANGES_REQUESTED** [review]

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts (hunk 3 above)
Comment (mrgrain) [reply packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts:26]: I think `il-central-1` also needs to be commented out.

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts (hunk 4 above)
Comment (SankyRed) [reply packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts]: Fixed! Thanks @mrgrain

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts (hunk 6 above)
Comment (SankyRed) [reply packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts]: Fixed!

File: packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts (hunk 8 above)
Comment (SankyRed) [reply packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts]: Fixed!

**Review by mrgrain — APPROVED** [review]

**Pull request merged**